clang/include/clang/AST/FormatString.h

```diff
@@ -417,6 +419,7 @@ class FormatSpecifier {
   ///  http://www.opengroup.org/onlinepubs/009695399/functions/printf.html
   bool UsesPositionalArg;
   unsigned argIndex;
+  unsigned size;
```

Comment:
I think it might be more clear to call this `ExplicitlyFixedSize` or something along those lines.